index.html

```diff
@@ -303,6 +303,32 @@ <h2 class="refer-link">LET'S CONNECT</h2>
                 </div>
             </div>
         </section>
-       </main>
+        <section id="contact-container">
```

Comment:
- [ ] Kindly adjust the background image to look similar to the Figma design, you can adjust the [background size](https://www.w3schools.com/cssref/css3_pr_background-size.asp) of the image 💪🏿